packages/web-components/src/components/cbp-list/cbp-list.stories.tsx

```diff
@@ -0,0 +1,103 @@
+export default {
+    title: 'Components/List',
+    tags: ['autodocs'],
+    argTypes: {
+        tag: {
+          control: 'select',
+          description: 'Type of List',
+          options: ['ul', 'ol'],
+        },
+        accessibilityText: {
+          description: 'Support for accessibility text',
+          control: 'text',
+        },
+        context : {
+          control: 'select',
+          options: [ "light-inverts", "light-always", "dark-inverts", "dark-always"]
+        },
+        sx: {
+          description: 'Supports adding inline styles as an object of key-value pairs comprised of CSS properties and values. Values should reference design tokens when possible.',
+          control: 'object',
+        },
+    },   
+    args: {
+      tag: 'ul',
+    },
+};
+
+function generateItems(items) {
+    const html = items.map(({ content}) => {
+        return `<li>${content}</li>`;      
+    });
+    return html.join('');
+  }
+  
+const Template = ({listItems, tag, context, accessibilityText }) => {
+    return ` 
+    <cbp-list
+      ${tag ? `tag=${tag}` : ''}
+      ${context && context != 'light-inverts' ? `context=${context}` : ''}
+      ${accessibilityText ? `accessibilityText=${accessibilityText}` : ''}
```

Comment:
this isn't working because it needs to be dash-case in HTML/template literal format. Just the HTML part - not the  ternary check.

Reply:
removed ternary check here